atomix/cluster/src/test/java/io/atomix/raft/roles/PassiveRoleTest.java

```diff
@@ -83,8 +79,8 @@ public void setup() {
   @Test
   public void shouldRejectRequestIfDifferentNumberEntriesAndChecksums() {
     // given
-    final List<RaftLogEntry> entries = generateEntries(1);
-    final AppendRequest request = new AppendRequest(2, "", 1, 1, entries, List.of(1L, 2L), 1);
+    final List<PersistedRaftRecord> entries = generateEntries(1);
+    final AppendRequest request = new AppendRequest(2, "", 1, 1, entries, 1);
```

Comment:
This test no longer makes sense since it's testing for a mismatch between entries and checksums in the append request

Reply:
Removed this test, added the other ignored test that verifies incorrect checksums.